cmd/lambda/accounts/delete.go

```diff
@@ -101,7 +101,7 @@ func destroyIAMPrincipal(account *db.Account) {
 		PolicyArn: fmt.Sprintf("arn:aws:iam::%s:policy/%s", account.ID, principalPolicyName),
 	})
 	// Log error, and continue
-	if err != nil {
+	if err != nil { //nolint
```

Comment:
This was a weird one. The linter claimed that `err` would _always_ be non-nil. Not sure why....

Reply:
Yea that is weird.  Maybe it gets tripped on that function returning multierrors and not error?